main.go

```diff
@@ -155,26 +159,19 @@ func runBackport(ctx context.Context, prArgs, commitArgs []string, release strin
 		return err
 	}
 
-	if release == "" {
-		release, err = getLatestRelease(ctx, c)
-		if err != nil {
-			return err
-		}
-	}
-
-	releaseBranch := "release-" + release
+	destBranch := getDestinationBranch(ctx, c, releaseArg, branchArg)
```

Comment:
Can you add a check to return an error if both "branch" and "release" are passed? Otherwise we may end up with obscure errors if someone passes conflicting values and the code silently uses one of them.